shared/test/fixtures/vcr/publicthumbnails/bad_channel_thumbnail.yml

```diff
@@ -1,61 +0,0 @@
----
```

Comment:
This test didn't go away, but thanks to the setup simplification it now makes no network requests, so it doesn't require a VCR cassette file.